ignite/cmd/cmd.go

```diff
@@ -78,14 +78,14 @@ To get started, create a blockchain:
 		NewTools(),
 		NewDocs(),
 		NewVersion(),
-		NewPlugin(),
+		NewApp(),
 		NewDoctor(),
 	)
 	c.AddCommand(deprecated()...)
 
 	// Load plugins if any
 	if err := LoadPlugins(ctx, c); err != nil {
```

Comment:
@Pantani regarding this and other comments for changes in the PR source code from "plugin" to "apps", I left them as "plugin" because otherwise we go down the rabbit hole refactoring the code. We can of course refactor the whole source code but that would take longer so if we decide to do it I think it should be done in another PR.

The idea right now is to have the **user** facing part of the plugins renamed to apps, which means having the documentation, config file and CLI commands API using "app", while the source code keeps using "plugins". In the documentation it is explained that Ignite Apps are implemented using a plugin system based on Hashicorp plugins to make it clear to **app developers**.

So, step one, we change the **user** facing parts to app.
Step two, if we decide to go for it we refactor the source code, which affects **app developers**.

WDYT @Pantani?

Reply:
I will work on a full refactor in a different PR

Reply:
make sense, we can also change the code slowly